app/src/main/res/layout/activity_create_event.xml

```diff
@@ -61,36 +58,16 @@
 
         <Spinner
             android:id="@+id/spSkillLevel"
-            android:layout_width="@dimen/spinnerDim"
-            android:layout_height="@dimen/spinnerDim"
-            android:background="@android:drawable/btn_dropdown"
-            android:spinnerMode="dropdown"
-            app:layout_constraintEnd_toStartOf="@id/tvAllowPlusOnes"
-            app:layout_constraintStart_toEndOf="@+id/tvSkillLevel"
-            app:layout_constraintTop_toTopOf="@+id/tvSkillLevel" />
-
-
-        <Spinner
-            android:id="@+id/spAllowPlusOnes"
-            android:layout_width="@dimen/spinnerDim"
-            android:layout_height="@dimen/spinnerDim"
-            android:background="@android:drawable/btn_dropdown"
-            android:spinnerMode="dropdown"
-            app:layout_constraintEnd_toStartOf="@id/tvAllowSpectators"
-            app:layout_constraintStart_toEndOf="@+id/tvAllowPlusOnes"
-            app:layout_constraintTop_toTopOf="@+id/tvAllowPlusOnes" />
-
-        <Spinner
-            android:id="@+id/spAllowSpectators"
-            android:layout_width="@dimen/spinnerDim"
-            android:layout_height="@dimen/spinnerDim"
+            android:layout_width="40dp"
```

Comment:
It ends up just being something like this: 
<img width="1305" alt="image" src="https://user-images.githubusercontent.com/876020/180288595-63133747-6859-4f40-abd5-c1501f728ae6.png">

You'll have enough room to see the text on the box then.  You can make it shorter if it's too big.  

Also, the default one is ugly, you can use this to make it a little prettier: https://github.com/jaredrummler/MaterialSpinner